docs/Using-Fleet/REST-API.md

```diff
@@ -918,6 +918,7 @@ None.
     "jira": null
   },
   "mdm": {
+    "mdm_enabled_and_configured": false,
```

Comment:
Then this could be come `enabled_and_configured` and still be consistent with the device response.